### ISSUE TITLE
Unable to weave class with multiple indexers

### ISSUE THREAD

**Issue by ste-art** [ArxOne/MrAdvice#39]
Example code:
```C#
    [MyAdvice]
    class Foo
    {
        public object this[string bar]
        {
            get { return new object(); }
        }

        public object this[int bar]
        {
            get { return new object(); }
        }
    }
```
Gives error:

    Error 1 Fody/MrAdvice.Weaver: Error while weaving method 'System.Object MrAdviceBug.Foo::get_Item(System.Int32)': System.InvalidOperationException: Sequence contains more than one matching element
    at System.Linq.Enumerable.Single[TSource](IEnumerable`1 source, Func`2 predicate)
    at ArxOne.MrAdvice.Reflection.Groups.MethodReflectionNode.LoadParent()
    at ArxOne.MrAdvice.Reflection.Groups.ReflectionNode.<GetSelfAndAncestors>d__14.MoveNext()
    at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
    at System.Linq.Enumerable.<ReverseIterator>d__a0`1.MoveNext()
    at System.Linq.Enumerable.<ConcatIterator>d__71`1.MoveNext()
    at System.Linq.Enumerable.<SelectManyIterator>d__14`2.MoveNext()
    at System.Linq.Enumerable.<DistinctIterator>d__81`1.MoveNext()
    at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
    at ArxOne.MrAdvice.Weaver.AspectWeaver.WeaveIntroductions(MethodDefinition method, TypeDefinition adviceInterface, ModuleDefinition moduleDefinition, Types types)
    at ArxOne.MrAdvice.Weaver.AspectWeaver.WeaveMethod(ModuleDefinition moduleDefinition, MarkedNode markedMethod, TypeDefinition adviceInterface, Types types)	MrAdviceBug

**Comment by picrap** [comment]
Ouch! Thanks for reporting this.

**Comment by picrap** [comment]
Fixed in 0.10.2

**Comment by ste-art** [comment]
Thanks!
